service/misc_services.cc

```diff
@@ -277,16 +277,16 @@ future<> view_update_backlog_broker::on_change(gms::inet_address endpoint, const
         char* end_bound;
         for (auto* ptr : {&current, &max}) {
             *ptr = std::strtoull(start_bound, &end_bound, 10);
-            if (*ptr == ULLONG_MAX) {
-                return make_ready_future();;
+            if (errno == ERANGE) {
```

Comment:
For extra safety, you should clear the errno (set to 0) before calling `std::strtoull`. I don't think this function is required to set the errno to 0 on success - at least, it's not indicated on the [cppreference page](https://en.cppreference.com/w/cpp/string/byte/strtoul). If it succeeds and we had a polluted errno before the call we might accidentally decide that there was an error.

Reply:
Added